config/manifests/kustomization.yaml

```diff
@@ -1,4 +1,8 @@
 resources:
+- bases
 - ../default
 - ../samples
 - ../scorecard
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: default
```

Comment:
Why do we need this change? It works perfectly without it

Reply:
removed.